Assets/BowAndArrowImproved/Scripts/Enemies/BAAIEnemy.cs

```diff
@@ -29,15 +29,20 @@ private void OnEnable()
         PlayWarCry();
     }
 
-    public void Die(bool playSound)
+    public void Die(bool deathByArrow)
```

Comment:
change name to `killedByPlayer`
what is tomorrow it's a bomb? or a gun? then you'll need to rename this